keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -70,35 +72,33 @@ class ResourceActuator(
       }
 
       try {
-        val (desired, current) = plugin.resolve(resource)
+        val (resolvedResource, desired, current) = plugin.resolve(resource)
```

Comment:
Here's where that resolved resource comes in handy. Now we can pass the resolved resource, which will include information about artifacts, into event constructors.